README.md

```diff
@@ -45,34 +54,39 @@ Run `ClientApp.main()` to start the client and see it connecting to the server.
 ## What's Next
 
 This example is built on top of in-memory data storage, uses `io.spine.` package 
-and demonstrates a really simple RPC interaction. To take it closer to the production needs, 
+and demonstrates a really simple RPC interaction. To take it closer to the production needs,
 the following steps are suggested:
 
 ### Gradle Configuration Changes:
 
  * Update `*.gradle` files with the artifact attributes, that correspond to your project.
 
-### Suggested `model` Changes 
+### Suggested `model` Changes
```

Comment:
There must be a simple change to a model. Without surrounding waters. Add a field to a command, add one more command. Make it really simple.